src/components/widgets/TransactionGraphicWidget.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <Card :loading="loading" v-if="isWidgetShown">
+        <template #title>{{getNameByKey('transactionGraphic')}}</template>
+
+        <template #body>
+            <div class="body">
+                <TransferGraphic v-if="TransactionTypes[data.type] === 'Transfer'" v-bind="data" />
+                <AddressAliasGraphic v-if="data.type === 16974" v-bind="data" />
+                <MosaicAliasGraphic v-if="data.type === 17230" v-bind="data" />
+                <NamespaceRegistrationGraphic v-if="data.type === 16718" v-bind="data" />
+            </div>
+        </template>
+    </Card>
+</template>
+
+<script>
+import Constants from '@/config/constants'
+import Card from '@/components/containers/Card.vue'
+import TransferGraphic from '@/components/transaction-graphic/TransferGraphic.vue'
+import AddressAliasGraphic from '@/components/transaction-graphic/AddressAliasGraphic.vue'
+import MosaicAliasGraphic from '@/components/transaction-graphic/MosaicAliasGraphic.vue'
+import NamespaceRegistrationGraphic from '@/components/transaction-graphic/NamespaceRegistrationGraphic.vue'
+
+export default {
+  props: {
+    managerGetter: String
+  },
+
+  components: {
+    Card,
+    TransferGraphic,
+    AddressAliasGraphic,
+    MosaicAliasGraphic,
+    NamespaceRegistrationGraphic
+  },
+
+  data() {
+    return {
+      TransactionTypes: Constants.TransactionType
+    }
+  },
+
+  computed: {
+    isWidgetShown() {
+      return this.data.type === 16724 ||
```

Comment:
I thinking it will be good, 
directly import `transactionType` from `symbol-sdk` rather than hard coded the transaction type.

Reply:
The reason is to avoid using sdk inside the Graphic components. It may be an issue in components lib. But for the explorer should be fine. Changed